internal/otgutils/print.go

```diff
@@ -0,0 +1,83 @@
+package otgutils
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/ondatra"
+	otgtelemetry "github.com/openconfig/ondatra/telemetry/otg"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// timer prints time elapsed in ms since a given start time
+func timer(start time.Time, name string) {
+	elapsed := time.Since(start)
+	log.Printf("%s took %d ms", name, elapsed.Milliseconds())
+}
+
+// WatchFlowMetrics is displaying flow stats for the given timeout duration
+func WatchFlowMetrics(t *testing.T, otg *ondatra.OTG, c gosnappi.Config, opts *WaitForOpts) error {
```

Comment:
What's the purpose of this function? Is it to be used like this?

```
flow.Start(t)
otgutils.WatchFlowMetrics(t, otg, c, opts) // was: time.Sleep(flowDuration)
flow.Stop(t)
```

I'm concerned that this may be causing log spam.

Reply:
I've created these print.go functions to help logging different OTG statistics while the test is running. Of course these are optional and we can remove them at the test level.
Log spam could be avoided with specifying a polling interval as seen below.

```go
otgutils.WatchFlowMetrics(t, otg, c, &otgutils.WaitForOpts{Interval: 2 * time.Second, Timeout: trafficDuration})
```

Reply:
I think this particular one is for sleeping between traffic flow start and stop, and I'm okay with it because otherwise the test would just be sleeping. Maybe rename it to something like:

```
func RunFlow(t *testing.T, flow *whatever.Flow, duration time.Duration) {
  flow.Start(t)
  ... // Sleep for duration, optionally display stats if stderr is a terminal.
  flow.Stop(t)
}
```

To avoid log spam, only output the stats to stderr if stderr is a terminal.

Reply:
I have removed the stats from the test .. the changes will be reflected in [PR#240](https://github.com/openconfig/featureprofiles/pull/240), and I will only display them once before doing the checks. 
I've also renamed the functions to PrintFlowMetrics and PrintPortMetrics